proxysql/README.md

```diff
@@ -14,6 +14,18 @@ The ProxySQL integration is included in the [Datadog Agent][3] package, so you d
 
 ### Configuration
 
+#### Enabling SSL
+To connect to ProxySQL using full SSL/TLS validation, enable the `tls_verify` option in `conf.yaml`. Include certificates and passwords needed to connect via SSL/TLS.
+
+```yaml
+    tls_verify: true
+    tls_cert: cert.pem
+    tls_ca_cert: ca_cert.pem
+    tls_private_key: private_key.pem
```

Comment:
Unfortunately, seems like ProxySQL doesn't support client-side certificates _yet_: https://github.com/sysown/proxysql/issues/1997

Probably worth not documenting that part and removing the extra configuration from the conf.yaml for now